app/controllers/providers/about_the_financial_assessments_controller.rb

```diff
@@ -9,8 +9,10 @@ def show
     def update
       return continue_or_draft if draft_selected?
 
-      if ready_for_citizen_to_enter_financial_details?
-        legal_aid_application.await_applicant!
+      unless legal_aid_application.applicant_entering_means?
```

Comment:
Can we have a chat about this? 
Multiple unless blocks seem a bit of a code smell!

Reply:
I forgot you would be off later in the day ...I have converted the nested unless block into a single statement which might be slightly better - but let's chat about it tomorrow :)